automations/python/workflows/generate_report.py

```diff
@@ -29,8 +39,10 @@
             "text": {
                 "type": "mrkdwn",
                 "text": (
-                    f"<{server_url}/{repository}/actions/runs/{run_id}|"
-                    "Click here to review the completed CI + CD workflow>."
+                    f"Hi {user_mention}, some CI + CD checks failed for your merge of <{commit_url}|{commit_message}>'.\n"  # noqa: E501
```

Comment:
There's a lonely `'` here before the newline at the end of this string. Should it be removed?

```suggestion
                    f"Hi {user_mention}, some CI + CD checks failed for your merge of <{commit_url}|{commit_message}>.\n"  # noqa: E501
```